src/hydro/srcterms/tabular_cooling.cpp

```diff
@@ -215,9 +215,10 @@ void TabularCooling::SrcTerm(MeshData<Real> *md, const Real dt) const {
 }
 
 template <typename RKStepper>
-void TabularCooling::SubcyclingFixedIntSrcTerm(MeshData<Real> *md, const Real dt,
+void TabularCooling::SubcyclingFixedIntSrcTerm(MeshData<Real> *md, const Real dt_,
                                                const RKStepper rk_stepper) const {
 
+  const auto dt = dt_; // capturing parameters still broken with Cuda 11.6 ...
```

Comment:
This change will likely conflict with my changes in the AGN Triggering PR.


Also, I'm a fan of explicitly labeling hacks for compilers as such so that future readers know that it's not the ideal code.
```suggestion
void TabularCooling::SubcyclingFixedIntSrcTerm(MeshData<Real> *md, const Real dt_,
                                               const RKStepper rk_stepper) const {

  const auto dt = dt_; //HACK capturing parameters still broken with Cuda 11.6 ...
```

Reply:
I'll update this when I fix the python format